src/fileviews/ucolumnsfileview.pas

```diff
@@ -743,15 +743,20 @@ procedure TColumnsFileView.UpdateFooterDetails(AInfo: Boolean);
   begin
     AFile:= CloneActiveFile;
     if Assigned(AFile) then
-    try
-      if AFile.IsNameValid then
-        lblInfo.Caption := AFile.Name
-      else if not AInfo then begin
-        inherited UpdateInfoPanel;
+      try
+        if AFile.IsNameValid then begin
+          if AFile.IsDirectory then 
```

Comment:
It should be `if gDirBrackets and (AFile.IsDirectory or AFile.IsLinkToDirectory) then` as I mentioned above. It should has same display name as name on the file panel.

Reply:
Yeah, sorry for missing that. Fixed.